deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -39,3 +42,25 @@ def callback_detect_vulnerability_detector_enabled(line):
     match2 = re.match(msg, line)
 
     return match1 is not None and match2 is None
+
+
+def make_vuln_callback(pattern):
+    """
+    Creates a callback function from a text pattern.
+
+    It already contains the vulnerability-detector prefix.
+
+    Parameters
+    ----------
+    pattern : str
+        String to match on the log
+
+    >>> callback_bionic_update_started = make_callback("Starting Ubuntu Bionic database update")
+    """
+    pattern = r'\s+'.join(pattern.split())
+    regex = re.compile(r'{}{}'.format(vulnerability_detector_prefix, pattern))
+
+    def _callback(line):
+        match = regex.match(line)
+        return match is not None
+    return _callback
```

Comment:
This is just a suggestion, up to you if you decide to keep it or change it, but this can be simplified by using this:
```suggestion
    return lambda line: regex.match(line) is not None
```